stytch/b2c/session.go

```diff
@@ -66,33 +64,3 @@ type Claims struct {
 	StytchSession SessionClaim `json:"https://stytch.com/session"`
 	jwt.RegisteredClaims
 }
-
-// Validation options in GoJWT are currently unexported. Once they're exported, we
-// can define this as a Valid() function, see
-// https://github.com/golang-jwt/jwt/blob/1096e506e671d6d6fe134cc997bbd475937392c8/validator_option.go#L9-L11 //nolint:lll
-func (c Claims) IsValid(projectID string) error {
```

Comment:
Just to check - did you mean to delete this function?

Reply:
No, I didn't, thank you for noticing! I've reimplemented and marked it as deprecated in the newest version of that commit: a7f164afb719124e566808f496e02c2b5c21e05d